api/customErrors.js

```diff
@@ -20,7 +38,7 @@ function CustomError() {
 CustomError.prototype = new Error();
 
 function ValidationError(errors) {
-  this.name = "validation";
+  this.name = "Validation";
```

Comment:
`this.name` gets used to set `errorType` in the JSON returned for the error. `errorType` is for the client code to examine and so should remain lower case. Can you arrange it so that `errorType` stays lower case please?

Reply:
Ok, should I change this.name = "Client" to "client" later in the file (line 77)? Or just leave everything how it was.

On the error toString, i can: hard-code the string ("Validation error[%d]..."), change it to upper-case (this.name.charAt(0).toUpperCase() + this.name.slice(1)) or just let it stay lower-case.

Reply:
Done